bibliopixel/main/devices.py

```diff
@@ -48,3 +48,5 @@ def run(args, settings):
 
 def set_parser(parser):
     parser.set_defaults(run=run)
+    parser.add_argument('--hardware-id', default='1D50:60AB', type=str)
```

Comment:
See above comments, but now I'm thinking that you might want to extract a tiny module - `hardware.py` - that had two methods: `def set_parser(parser):` which is the last two lines of this;  and `make_devices(args)` which just returns `Devices(args.hardware_id, args.baud)`.

_If_ you think that there will be a third command line function that will use devices, you should do it.  If you don't think so, you shouldn't.